sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/implementation/BlobServicePropertiesImpl.java

```diff
@@ -195,4 +208,24 @@ public BlobServicePropertiesImpl withContainerDeleteRetentionPolicyDisabled() {
         this.innerModel().withContainerDeleteRetentionPolicy(new DeleteRetentionPolicy().withEnabled(false));
         return this;
     }
+
+    @Override
+    public BlobServicePropertiesImpl withLastAccessTimeTrackingPolicyEnabled() {
+        if (this.innerModel().lastAccessTimeTrackingPolicy() == null) {
+            this.innerModel().withLastAccessTimeTrackingPolicy(new LastAccessTimeTrackingPolicy());
+        }
+        this.innerModel().lastAccessTimeTrackingPolicy().withEnable(true);
+        return this;
+    }
+
+    @Override
+    public BlobServicePropertiesImpl withLastAccessTimeTrackingPolicy(LastAccessTimeTrackingPolicy policy) {
+        this.innerModel().withLastAccessTimeTrackingPolicy(policy);
+        return this;
+    }
+
+    @Override
+    public BlobServicePropertiesImpl withLastAccessTimeTrackingPolicyDisabled() {
+        return withLastAccessTimeTrackingPolicy(new LastAccessTimeTrackingPolicy().withEnable(false));
```

Comment:
Should we do as in `withLastAccessTimeTrackingPolicyEnabled`? Aka, if policy exist, only change the value of enable?

Reply:
Despite we only change the enable to false, the whole `lastAccessTimeTrackingPolicy` will be null. 

But why not, maybe the behavior will change in the future..

Reply:
OK, I see. That might be service behavior (if disabled, the whole object is no longer needed). But I guess if possible we still make client logic consistent (if backend is fine with it).

Reply:
Sure.

Reply:
Changed.